articles/azure-app-configuration/enable-dynamic-configuration-dotnet-core-push-refresh.md

```diff
@@ -41,7 +41,9 @@ In this tutorial, you learn how to:
 
 ## Prerequisites
 
-To do this tutorial, install the [.NET Core SDK](https://dotnet.microsoft.com/download).
+* Install [.NET Core SDK](https://dotnet.microsoft.com/download).
+* Quickstart: [Create a .NET Core app with App Configuration](./quickstart-dotnet-core-app.md)
```

Comment:
Now that I read this doc again, I think we should put the tutorial below as the prereq instead. Not the quickstart because the quickstart doesn't talk about config refresh at all. Can you please make the same change for line 31?

Tutorial: \[Use dynamic configuration in a .NET Core app](./enable-dynamic-configuration-dotnet-core)

Reply:
So then remove the quickstart?

Reply:
Change to the tutorial linked above.

Reply:
updated